.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -1,6 +1,10 @@
 ---
 name: Bug report
 about: Create a report to help us improve
+title: ''
```

Comment:
How are these fields used?

Reply:
@jag-sfdc - This is the template users see when they create a new issue using the "bug" template - https://github.com/forcedotcom/salesforcedx-vscode/issues/new?assignees=&labels=&template=Bug_report.md

the change is to ask for the SF extension version vs. the VS Code version

Reply:
yes, I got that part. I didn't understand what these "title", "labels" and "assignees" fields are.

Reply:
not sure.  i think it's just a standard part of VS Code templates for issues?  i didn't change anything but the one label...

Reply:
@brpowell - Do you know what these fields are for? When the markdown is rendered, they appear as a table.
![Screen Shot 2021-06-30 at 9 14 45 AM](https://user-images.githubusercontent.com/58611665/123995999-bc6aef80-d983-11eb-9d93-f46703776fea.png)

Reply:
I believe they are a mechanism to automatically apply labels/assignees based on the template. If they weren't on purpose Stephanie you can remove them.

Reply:
I think this is auto-added frontmatter https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

we can keep this.